homeassistant/helpers/customize.py

```diff
@@ -1,25 +1,51 @@
 """A helper module for customization."""
 import collections
-from typing import Dict, List
+from typing import Any, Dict, List
 import fnmatch
+import voluptuous as vol
 
 from homeassistant.const import CONF_ENTITY_ID
 from homeassistant.core import HomeAssistant, split_entity_id
+import homeassistant.helpers.config_validation as cv
 
-_OVERWRITE_KEY = 'overwrite'
-_OVERWRITE_CACHE_KEY = 'overwrite_cache'
+_OVERWRITE_KEY_FORMAT = '{}.overwrite'
+_OVERWRITE_CACHE_KEY_FORMAT = '{}.overwrite_cache'
 
+_CUSTOMIZE_SCHEMA_ENTRY = vol.Schema({
+    vol.Required(CONF_ENTITY_ID): vol.All(
```

Comment:
Ignore this, forgot that this is a convoluted list of entity_ids, domains and globs for a moment...